google/cloud/security/inventory/pipeline_configs.yaml

```diff
@@ -0,0 +1,40 @@
+
+- resource: bigquery_datasets
+  should_run: true
+
+- resource: buckets
+  should_run: false
+
+- resource: buckets_acls
+  depends_on: buckets
+  should_run: true
+
+- resource: cloudsql
+  should_run: true
+
+- resource: firewall_rules
+  should_run: true
+
+- resource: folders
+  should_run: false
+
+- resource: forwarding_rules
+  should_run: true
+
+- resource: group_members
+  should_run: true
+
+- resource: groups
+  should_run: false
+
```

Comment:
If this is also going to be our default config. Should we just comment out false resources (but also change them to enabled)